devel/code_generation.py

```diff
@@ -18,7 +18,7 @@ def main():
     for output_fname in output_fnames:
         process_template(output_fname, template_kwargs=dict(extensions=extensions))
     
-    create_md_ts_files('.')
+    create_text_ts_files('.')
```

Comment:
The reason for this change: I needed to import the text of .py files in addition to .md files -- this is for the snippets displayed on the gui used to import data into the system.